### PR TITLE
[LW-10266] Upgrade Axios version to 0.28.0

### PR DESCRIPTION
**Pull request by VanessaPC**
Upgrade the following packages in the cardano-js-sdk to use the latest version of Axios (0.28.0. 
Packages affected: `cardano-services`, `cardano-services-client`, `e2e` and `util-dev`. 

# Context
Lace wants to upgrade to use the latest of Axios and it's blocked because types changes in this dependency won't allow.

# Proposed Solution

# Important Changes Introduced
`cardano-services`, `cardano-services-client`, `e2e` and `util-dev` now use Axios 0.28.0 .

### CODE REVIEW

**Review by iccicci — APPROVED** [review]
LGTM!

**Review by rhyslbw — APPROVED** [review]

**Comment by VanessaPC** [comment]
@rhyslbw @iccicci tests are passing now, would you like to re-review/re-approve?

File: packages/cardano-services/test/Handle/HandleHttpService.test.ts [hunk 1]
```diff
@@ -56,7 +56,7 @@ describe('HandleHttpService', () => {
 
       const { message, response } = error;
 
-      if (!response) return { message, statusText: error.status };
+      if (!response) return { message, statusText: `${error.status}` };
```
[the remark this answers is not in the document]
Comment (VanessaPC) [reply packages/cardano-services/test/Handle/HandleHttpService.test.ts]: addressed here: 2c726944a091c4ad2f489ab969539b1e996f066c

**Review by rhyslbw — APPROVED** [review]

**Review by pczeglik-iohk — APPROVED** [review]
excellent work @VanessaPC

**Pull request merged**